### DISCUSSION TITLE
blog/building-github-pages-with-hugo/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#352] [Announcements]
# blog/building-github-pages-with-hugo/

Hugo 是一种通用的网站框架，本问教你如何使用 Hugo 来构建静态网站。

https://jimmysong.io/blog/building-github-pages-with-hugo/

**Comment by smilelhong** [comment]
hello，博主的这个博客的主题用的哪个啊，感觉特别舒适

**Reply by rootsongjc** [reply]
这个网站是结合 [Educenter Hugo](https://github.com/themefisher/educenter-hugo) 和 [Hugo Blox](https://hugoblox.com/) 两个主题构建，并做了大量的优化，见[网站改版通知](https://jimmysong.io/notice/website-revamp-notice/)

**Comment by smilelhong** [comment]
非常感谢！
